Sources/SwifterSwift/MapKit/MKMapViewExtensions.swift

```diff
@@ -44,7 +44,25 @@ public extension MKMapView {
 
         return annotationView
     }
-
+    
+    /// SwifterSwift: Zooms in on multiple mapView coordinates.
+    ///
+    /// - Parameters:
+    ///   - coordinates: Gets the array of type CLLocationCoordinate2D.
+    ///   - meter: Takes the type of Double zoom.
+    ///   - insets: Gives space to the CGFloat type screen.
+    ///   - animated: The animation control takes the Boolean value.
```

Comment:
Please give clearer descriptions for the parameters, e.g.
> insets: The amount of additional space (measured in screen points) to make visible around the specified rectangle

(I stole that description from Apple's documentation for `setVisibleMapRect(_:edgePadding:animated:)` 😉)